CHANGELOG.md

```diff
@@ -4,8 +4,12 @@
 
 ## [Unreleased]
 
-### Breaking changes
+### New features
+- Add vertical alignement modifiers in `objects.media`. [[#379](https://github.com/inuitcss/inuitcss/issues/379)]
 
+### Breaking changes
+- Use flexbox in `objects.media`. [[#379](https://github.com/inuitcss/inuitcss/issues/379)]
```

Comment:
The fact that we use flexbox now is not a breaking change. If we do it well, the inuitcss shouldn’t notice the change at all.